neo4j/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM neo4j:3.5.11
+FROM neo4j:3.5.11-enterprise
```

Comment:
But that would mean that people cannot run neo4j without license keys anymore? :thinking:

Reply:
well, apparently they don't check the keys, you just have to agree to the license
see https://github.com/Human-Connection/Human-Connection/pull/1787/files#diff-4e5e90c6228fd48698d074241c2ba760R60

but also, we, since we are deploying to production, are agreeing to the commercial license and must provide the license number if asked, whereas others who are using it to help develop the site, would be agreeing to the developer's license https://neo4j.com/licensing/

but yeah, I think if people were wanting to set it up in different places/contexts, they might want to change this to the non-enterprise version

Reply:
Seriously? It just works out of the box and we don't have to setup our license keys or anything like that?

Reply:
it works with local docker dev... like if you run `docker-compose up` and go to localhost:7474 and log in, you can see in the bottom left that it says you are using `Enterprise` edition